internal/controllers/ingresses.go

```diff
@@ -100,8 +100,8 @@ func (r *Reconciler) createOrUpdateIngress(ctx context.Context, ingress *netv1.I
 	config *operatorv1beta1.NetworkConfiguration) (*netv1.Ingress, error) {
 	op, err := controllerutil.CreateOrUpdate(ctx, r.Client, ingress, func() error {
 		// Set labels and annotations from CR
-		ingress.Labels = config.Labels
-		ingress.Annotations = config.Annotations
+		mergeLabelsAndAnnotations(&ingress.ObjectMeta, config.Labels, config.Annotations)
```

Comment:
I think we need to be extra careful here as `config.Labels` and `config.Annotations` can be `nil`. We will need the same handlers as in Services and Routes.

https://github.com/cryostatio/cryostat-operator/blob/2d386930dc96f0dcaf937987ec35874006c53b61/internal/controllers/services.go#L228